pulpcore/cli/ansible/content.py

```diff
@@ -11,7 +15,6 @@
 )
 from pulpcore.cli.common.generic import (
     GroupOption,
-    chunk_size_option,
```

Comment:
Please explain...

Reply:
The collection upload wasn't using it (like it actually didn't do anything even if you specified it).  I can change this to use `allow_with_context` to better reflect its use only with roles.